backend/src/protocol/request.rs

```diff
@@ -14,12 +14,18 @@ pub struct AddFeed {
     pub url: String,
 }
 
+#[derive(Deserialize, Debug)]
+pub struct MarkRead {
```

Comment:
Yeah this should definitely be a SetReadStatus with a bool for whether to mark read. It might make sense to make it a batch API as well, not sure.

Reply:
In the context of "mark a specific entry as read" a batch API doesn't make much sense, but now that I've had time to think about it, a batch API could be useful for a feature like "mark this entire feed as read".